docs/serving/services/_index.md

```diff
@@ -1,6 +1,7 @@
 ---
-title: "Knative Services"
-linkTitle: "Services"
-weight: 10
+title: "Knative services"
+weight: 02
 type: "docs"
 ---
+
+Knative services are used to deploy an application. Each Knative service is defined by a route and a configuration, which have the same name as the service, and are contained in a YAML file. Each time the configuration is updated, a new revision is created. Revisions are each backed by a deployment with two [Kubernetes Services](../knative-kubernetes-services).  
```

Comment:
The "Revisions are backed by a deployment with two K8s services" seems like an implementation detail.

Maybe "Revisions are immutable snapshots of a particular configuration, and use underlying Kubernetes resources to scale the number of Pods based on traffic." ?